server/views/partials/assessment/tasks.njk

```diff
@@ -1,8 +1,40 @@
 {% from "govuk/components/tabs/macro.njk" import govukTabs %}
+{%- from "moj/components/ticket-panel/macro.njk" import mojTicketPanel -%}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+
+{%  macro ticket(params) %}
+  <h2 class="govuk-heading-m govuk-!-margin-bottom-2">{{params.name}}</h2>
+  {{ govukButton({
+    text: params.action,
+    href: params.href,
+    classes: "govuk-!-margin-bottom-1"
+  }) }}
+{% endmacro %}
```

Comment:
put in the first task:

<kbd>
<img width="900" alt="Screenshot 2024-09-26 at 08 43 34" src="https://github.com/user-attachments/assets/d8e7b73f-3a16-4b98-b0f7-0da302ca11fc">
</kbd>